templates/trader.yaml

```diff
@@ -1,8 +1,8 @@
 name: "Trader Agent"
 description: "A single-agent service (sovereign agent) placing bets on Omen"
-hash: bafybeicbfnqfc3potbfmvrmssaxepwilipe5myjbg4xh7ix2sdll4kyc6m
+hash: bafybeidts6y6wytfcwlhnvnacf3vhm6lkiocoe22kkmxxx2onzguez3y3q
 image: https://operate.olas.network/_next/image?url=%2Fimages%2Fprediction-agent.png&w=3840&q=75
-service_version: v0.18.1
+service_version: v0.18.2
```

Comment:
@0xArdi Confirm this version is OK. Version 0.19.0 is for multi-bet strategy, but was downgraded in QS.

Reply:
As discussed with @dagacha , version should be v0.18.2